codalab/rest/interpret.py

```diff
@@ -158,6 +158,32 @@ def fetch_interpreted_worksheet(uuid):
     - resolve_interpreted_items: get more information about a worksheet.
     In the future, for large worksheets, might want to break this up so
     that we can render something basic.
+    Return: 
+        worksheet_info dict{}:
+            key:[value_type] <description>
+            items:[list] resolved BLOCK items from raw_items, 
+                            bundles will be grouped into table block items, text items might be grouped into one markdown block
+                            TODO: This should really be renamed to block_items to avoid confusion, 
+                                    but too many places are using 'item'
+                                    and its not easy to tell if they all mean the same
+            source-list:[list] source lines
+            raw_to_block-list:[list] raw_item to its block index pair 
```

Comment:
If we've renamed "raw" to "source", then we might want to replace "raw_to_block" to "source_to_block", etc.

Reply:
raw_to_block is actually raw items to block, not source

Reply:
by raw items, it means items rendered by the source but not grouped into block yet